### PR TITLE
gacutil: add page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/gacutil.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# gacutil
+
+> Global Assembly Cache management utility.
```
Comment (JakobMiksch) [inline pages/common/gacutil.md:3]:
```suggestion
> Global Assembly Cache (CAG) management utility.
```

**Review by blueskyson — APPROVED** [review]

File: pages/common/gacutil.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# gacutil
+
+> Global Assembly Cache (CAG) management utility.
+> More information: <https://manned.org/gacutil.1>.
```
Comment (mfrw) [inline pages/common/gacutil.md:4]: Can we please drop the trailing `.1`
```suggestion
> More information: <https://manned.org/gacutil>.
```

**Review by mfrw — COMMENTED** [review]
A teeny nit.

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks

**Pull request merged**